src/fundus/publishers/fr/euronews.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute, utility
+
+
+class EuronewsParser(ParserProxy):
+    class V1(BaseParser):
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.meta.get("og:title")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            summary_selector = CSSSelector("p.c-article-summary")
+            subheadline_selector = CSSSelector("div.c-article-content > h2")
+            paragraph_selector = CSSSelector("div.c-article-content > p")
```

Comment:
Those should be added as class variables of the parser so they won't be compiled every time `body` is called.

Please refer to another parser to see how to add them as class variables.

Reply:
Sure, I’ll do it tonight.